hare/flows_test.go

```diff
@@ -573,7 +440,10 @@ func Test_multipleCPsAndIterations(t *testing.T) {
 	for _, out := range outputs {
 		for lid := types.GetEffectiveGenesis().Add(1); !lid.After(finalLyr); lid = lid.Add(1) {
 			require.NotNil(t, out[lid])
-			require.ElementsMatch(t, pList[lid], out[lid].Proposals)
+			require.ElementsMatch(t, types.ToProposalIDs(pList[lid]), out[lid].Proposals)
 		}
 	}
+	for _, h := range test.hare {
+		h.Close()
+	}
```

Comment:
Does this need to happen independently of if the test completes or not? If yes we could use a `t.Cleanup` or `defer` for `h.Close()`.